File: EcoParqueoUAM_Service/src/main/java/org/uam/ecoparqueouam_service/model/RegistroAcceso.java
package org.uam.ecoparqueouam_service.model;

import jakarta.persistence.Entity;
import jakarta.persistence.Table;

@Entity
@Table(name="registro_acceso")
public class RegistroAcceso extends BaseEntity {
    private String placa;
    private String parqueoNombre;
    private Long fechaHora;

    public String getPlaca() {
        return placa;
    }

    public void setPlaca(String placa) {
        this.placa = placa;
    }

    public String getParqueoNombre() {
        return parqueoNombre;
    }

    public void setParqueoNombre(String parqueoNombre) {
        this.parqueoNombre = parqueoNombre;
    }

    public Long getFechaHora() {
        return fechaHora;
    }

    public void setFechaHora(Long fechaHora) {
        this.fechaHora = fechaHora;
    }
}

File: EcoParqueo-UAM/app/src/main/java/com/uam/ecoparqueo/model/RegistroAcceso.kt
package com.uam.ecoparqueo.model

import com.google.gson.annotations.SerializedName

data class RegistroAcceso(
    @SerializedName("id") val id: String? = null,
    @SerializedName("placa") val placa: String,
    @SerializedName("parqueoNombre") val parqueoNombre: String,
    @SerializedName("fechaHora") val fechaHora: Long
)
